blockchain/common_test.go

```diff
@@ -100,6 +104,34 @@ func createTestDatabase(dbName string, dbType string, net wire.CurrencyNet) (dat
 	return db, teardown, nil
 }
 
+// createTestUtxoDatabase creates a test UTXO database with the provided
+// database name.
+func createTestUtxoDatabase(dbName string) (*leveldb.DB, func(), error) {
+	// Construct the database filepath and remove all from that path.
+	dbPath := filepath.Join(os.TempDir(), dbName)
+	_ = os.RemoveAll(dbPath)
+
+	// Open the database (will create it if needed).
+	opts := opt.Options{
+		Strict:      opt.DefaultStrict,
+		Compression: opt.NoCompression,
+		Filter:      filter.NewBloomFilter(10),
+	}
+	db, err := leveldb.OpenFile(dbPath, &opts)
+	if err != nil {
+		return nil, nil, err
+	}
+
+	// Setup a teardown function for cleaning up.  This function is returned to
+	// the caller to be invoked when it is done testing.
+	teardown := func() {
+		db.Close()
+		os.RemoveAll(dbPath)
```

Comment:
Above you did `_ = os.RemoveAll(dbPath)` which I like. Could also do for both of these so it's easy to see the error is ignored. Here and in similar functions.

Reply:
Agreed, it's nice to see that an error is being ignored, even for tests.  Updated.